### PR TITLE
User Experience

### PR DESCRIPTION
**Pull request by mduddukuru**
I've created the front end of the User Experience Form, under Blog as a dropdown! 
The implementation of the form is done, but the response page and back-end work will need to be done further. 
Thank you, 
The only changes are in the:
models.py
urls.py
views.py
forms.py
navigation.html - includes
created a new feedback.html

### CODE REVIEW

File: lastupdate.html
Comment (s-bonnor) [inline lastupdate.html:1]: this doesnt look like it belongs

File: test.html
Comment (s-bonnor) [inline test.html:1]: this also doesnt look like it belongs

**Review by s-bonnor — CHANGES_REQUESTED** [review]
Could you explain the purpose of the 2 html files in the root folder

File: lastupdate.html
Comment (mduddukuru) [reply lastupdate.html:1]: Yes actually, they don't belong, but I wasn't sure how to delete those changes.

File: test.html
Comment (mduddukuru) [reply test.html:1]: Yes actually, they don't belong, but I wasn't sure how to delete those changes.

**Review by s-bonnor — APPROVED** [review]
All good

**Pull request merged**